radio/src/targets/common/arm/stm32/bootloader/boot.cpp

```diff
@@ -524,15 +524,24 @@ int main()
     }
 
     if (state == ST_REBOOT) {
-      lcdClear();
-      lcdRefresh();
-      lcdRefreshWait();
 
 #if defined(RTC_BACKUP_RAM)
       rtcInit();
       RTC->BKP0R = SOFTRESET_REQUEST;
 #endif
 
+#if defined(RADIO_COMMANDO8)
+      lcdClear();
+      lcdDrawText(2, 22,"Press the power button.");
+      lcdDrawText(2, 33,"Exit the flashing mode.");
+      lcdRefresh();
+      lcdRefreshWait();
+      while(1);
```

Comment:
See discussion here: https://github.com/opentx/opentx/pull/8905#discussion_r851485381

This code should be best replaced by a single function called `blExit()` which is implemented differently in the target's `bootmenu.cpp`. There you could place `#if defined(...)` and replace the default taranis implementation.